src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -192,8 +188,6 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
 
         String body = format(
             CONFIRMATION_SUMMARY,
-            format("/cases/case-details/%s#CaseDocuments", caseData.getCcdCaseReference()),
-            claimIssueConfiguration.getResponsePackLink(),
             formattedServiceDeadline
```

Comment:
We will work out what the actual deadline in another story (so we are good here)